### PR TITLE
Loosen spacy requirements to less than 3.1.0 and more than 3.0.0

### LINKED ISSUE #243: Spacy dependency conflict with SciSpacy and is too strict
Our current requirement for spacy is `spacy[lookups]==3.0.0` which is too strict and it creates a conflict with scispacy which asks for >3.0.1 due to a possible fix that is introduced. We can definitely loosen the dependency slighly without breaking anything hopefully.

### PR DESCRIPTION
**Pull request by nsorros**
Description
---

Fixes #243

Checklist
---

- [x] Added link to Github issue or Trello card
- [ ] Added tests

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
         'boto3',
         'scikit-learn',
         'scipy==1.4.1',
-        'spacy[lookups]==3.0.0',
+        'spacy[lookups]<3.1.0,>=3.0.0',
```
Comment (aCampello) [inline setup.py:49]: In the issue description, it says that scispacy which asks for `>3.0.1.`. Do you think just `> 3.0.1` wouldn't work?

**Review by aCampello — APPROVED** [review]
It all passes, go ahead and merge! But I was wondering whether we can relax even more ?

File: setup.py (hunk 1 above)
Comment (nsorros) [reply setup.py:49]: It would but I guess we should set an upper bound, should we say <4.0.0?

Comment (aCampello) [reply setup.py:49]: Yeah, that'd be more relaxed!

Comment (nsorros) [reply setup.py]: aa2dc2c

**Pull request merged**